module/VuFind/src/VuFind/Controller/CoverController.php

```diff
@@ -115,6 +125,24 @@ protected function getImageParams()
         ];
     }
 
+    /**
+     * Is the provided URL included on the configured allow list?
+     *
+     * @param string $url URL to check
+     *
+     * @return bool
+     */
+    protected function proxyAllowedForUrl(string $url): bool
+    {
+        $host = parse_url($url, PHP_URL_HOST);
```

Comment:
Needs a check that result is not false in case of a malformed URL.

Reply:
Good point! Added.